src/Rings.jl

```diff
@@ -227,7 +227,8 @@ is_zero(R::NCRing) = is_trivial(R)
 @doc raw"""
     is_perfect(F::Field)
 
-Test whether the field $F$ is perfect.
+Test whether the field $F$ is perfect, that is, whether the characteristic is zero or
+$F^p = F$, where $p > 0$ is the characteristic of $F$.
```

Comment:
I find the notation $F^p$ ambiguous. How about
```suggestion
else whether every element of $F$ admits a $p$-th root, where $p > 0$ is the characteristic of $F$.
```

Reply:
Fine with me. Out of curiosity, where is the ambiguity?

Or maybe
```
or the characteristic is positive and the Frobenius endomorphism of $F$ is surjective?
```
This would not need introducing `p`.

Reply:
`F^p` could either be a dim p F-vectors pace or the element-wise exponentiation

Reply:
I am also fine stating it with the Frobenius